tools/pd-ctl/README.md

```diff
@@ -123,52 +125,27 @@ Usage:
     "patrol-region-interval": "100ms",
     "region-schedule-limit": 2048,
     "replica-schedule-limit": 64,
-    "scheduler-max-waiting-operator": 3,
-    "schedulers": {
-      "balance-hot-region-scheduler": "null",
-      "balance-leader-scheduler": "null",
-      "balance-region-scheduler": "null",
-      "label-scheduler": "null"
-    },
-    "schedulers-v2": [
-      {
-        "args": null,
-        "args-payload": "",
-        "disable": false,
-        "type": "balance-region"
-      },
-      {
-        "args": null,
-        "args-payload": "",
-        "disable": false,
-        "type": "balance-leader"
-      },
-      {
-        "args": null,
-        "args-payload": "",
-        "disable": false,
-        "type": "hot-region"
-      },
-      {
-        "args": null,
-        "args-payload": "",
-        "disable": false,
-        "type": "label"
-      }
-    ],
+    "scheduler-max-waiting-operator": 5,
     "split-merge-interval": "1h0m0s",
     "store-balance-rate": 15,
+    "store-limit-mode": "manual",
     "tolerant-size-ratio": 0
   }
 }
 >> config show all                            // Display all config information
 >> config show replication                    // Display the config information of replication
 {
   "max-replicas": 3,
-  "location-labels": ""
+  "location-labels": "",
+  "strictly-match-label": "false",
+  "enable-placement-rules": "false"
 }
 >> config show cluster-version                // Display the current version of the cluster, which is the current minimum version of TiKV nodes in the cluster and does not correspond to the binary version.
-"2.0.0"
+"4.1.0-alpha"
+
+>> config delete namespace <name> [flags]
```

Comment:
We have already remove namespace

Reply:
ok, and we need to delete it from help echo